config/locales/en.yml

```diff
@@ -73,7 +73,7 @@ en:
       correspondence:
         one: Correspondence
         other: Correspondences
-      detailed_guidance:
```

Comment:
Yes, you're right. I quickly substituted them, we can go back eventually. In Whitehall I found both terms used, although "detailed_guide" seems to be used more frequently.